crates/bevy_pbr/src/render_graph/forward_pipeline/forward.frag

```diff
@@ -244,15 +238,12 @@ void main() {
 #endif
 
 #ifndef STANDARDMATERIAL_UNLIT
-    // calculate non-linear roughness from linear perceptualRoughness
-    float roughness = perceptualRoughnessToRoughness(perceptual_roughness);
-
     vec3 N = normalize(v_Normal);
     vec3 V = normalize(CameraPos.xyz - w_Position.xyz);
 
-    // Remapping [0,1] reflectance to F0
+    // Reflectance is remapped before being passed to the GPU
     // See https://google.github.io/filament/Filament.html#materialsystem/parameterization/remapping
-    vec3 F0 = 0.16 * reflectance * reflectance * (1.0 - metallic) + output_color.rgb * metallic;
+    vec3 F0 = reflectance * (1.0 - metallic) + output_color.rgb * metallic;
```

Comment:
FYI, this can be rewritten as `vec3 F0 = mix(reflectance, output_color.rgb, metallic);`